ClangPowerTools/ClangPowerTools/Commands/CommandController.cs

```diff
@@ -392,6 +393,15 @@ public void OnLicenseChanged(object sender, LicenseEventArgs e)
       activeLicense = e.IsLicenseActive;
     }
 
+    public void VisibilityOnBeforeCommand(object sender, EventArgs e)
+    {
+      if (!(sender is OleMenuCommand command))
```

Comment:
Use standard boolean comparison for `if` statements

Reply:
Ok, I will do so.